apps/timetable/admin/index.html

```diff
@@ -0,0 +1,150 @@
+                </div>
+                <main id="gh-main"><!-- --></main>
+            </div>
+            <div class="clearfix"></div>
+        </div>
+
+        <script id="gh-header-template" type="text/template">
+            <h1>Timetable Administration</h1>
+            <% if (gh.data.me) { %>
+                <form id="gh-signout-form" class="form-inline pull-right" role="form">
+                    <button type="submit" class="btn btn-default gh-btn-secondary">
+                        <i class="fa fa-sign-out"></i>Sign out
+                    </button>
+                </form>
+
+            <!--/* TODO: Add else clause that shows a login button (Raven login) */-->
+            <% } %>
+        </script>
+
+        <script id="gh-help-template" type="text/template">
+            <div id="gh-help">
+                <p>Need help? Contact: <a href="mailto:tthelp@cam.ac.uk" title="tthelp@cam.ac.uk">tthelp@cam.ac.uk</a></p>
+            </div>
+        </script>
+
+        <script id="gh-tripos-help-template" type="text/template">
+            <p class="gh-tripos-help">To start editing timetables choose a tripos from the list above, or select from your list below</p>
+        </script>
+
+        <script id="gh-login-template" type="text/template">
+            <div id="gh-login-form" class="pull-left">
+                <form class="gh-signin-form" role="form">
+                    <h2>Sign in</h2>
+                    <div class="form-group">
+                        <label for="gh-signin-email">Email address</label>
+                        <input type="username" name="username" class="form-control" id="gh-signin-email" placeholder="Enter your email" required>
+                    </div>
+                    <div class="form-group">
+                        <label for="gh-signin-password">Password</label>
+                        <input type="password" name="password" class="form-control" id="gh-signin-password" placeholder="Enter your password" required>
+                    </div>
+                    <button type="submit" id="gh-signin" class="btn btn-default gh-btn-primary">
+                        <i class="fa fa-sign-in"></i>Sign in
+                    </button>
+                </form>
+            </div>
+        </script>
+
+        <script id="gh-modules-template" type="text/template">
+            <%= _.partial('modules', {'data': data}) %>
+        </script>
+
+        <script id="gh-subheader-part-template" type="text/template">
+            <%= _.partial('subheader-part', {'data': data}) %>
+        </script>
+
+        <script id="gh-subheader-picker-template" type="text/template">
+            <%= _.partial('subheader-picker', {'data': data}) %>
+        </script>
+
+        <script id="gh-subheader-pickers-template" type="text/template">
+            <a class="pull-right gh-show-video" href="#" style="display: none;"><i class="fa fa-youtube-play"></i> Help video<a>
+            <%= _.partial('subheader-pickers') %>
```

Comment:
Would it make sense to combine the `subheader-pickers` partial with the two other partials?